tcms/management/management/commands/baseurl.py

```diff
@@ -0,0 +1,34 @@
+from django.core.management.base import BaseCommand
+from django.contrib.sites.models import Site
+
+
+class Command(BaseCommand):
+    help = "Sets the base URL of KiwiTCMS instance. " \
```

Comment:
Name is `Kiwi TCMS`.

If you use string concatenation across multiple lines you don't need a back slash.